external/storm-druid/pom.xml

```diff
@@ -40,16 +40,26 @@
             <groupId>io.druid</groupId>
             <artifactId>tranquility-core_2.11</artifactId>
             <version>0.8.2</version>
-        </dependency>
-        <dependency>
-            <groupId>io.druid</groupId>
-            <artifactId>druid-server</artifactId>
-            <version>0.9.1</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-server</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-core</artifactId>
+                </exclusion>
+                <exclusion>
+                  <groupId>mysql</groupId>
+                  <artifactId>mysql-connector-java</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.scala-lang</groupId>
             <artifactId>scala-library</artifactId>
             <version>2.11.8</version>
+            <scope>provided</scope>
```

Comment:
@HeartSaVioR  Thanks for the review.  Yes, Scala dependency can be provided by users. Updated the document.